libs/barista-components/experimental/quick-filter/src/quick-filter-group.scss

```diff
@@ -0,0 +1,22 @@
+@import '../../../core/src/style/variables';
+
+:host {
+  display: block;
+  padding: 8px 0;
+}
+
+.dt-quick-filter-group-headline {
+  margin: 0 0 8px;
+  font-size: 18px;
+  font-weight: normal;
+  color: $gray-700;
```

Comment:
size is measured from the invision prototypes

Reply:
Please do not trust the invision prototypes, please check with ux which styling it will need and make them aware that this does not align with any of our current sizes.